packages/lib/gpcircuits/src/list-membership.ts

```diff
@@ -0,0 +1,131 @@
+import { CircuitSignal } from "./types";
+import { BABY_JUB_NEGATIVE_ONE } from "@pcd/util";
+import { PODValue, podValueHash } from "@pcd/pod";
+import { poseidon1, poseidon2, poseidon3, poseidon4} from "poseidon-lite";
+
+const poseidon = [poseidon2, poseidon3, poseidon4];
+
+export type ListMembershipModuleInputs = {
+  valueHash: CircuitSignal;
+  list: CircuitSignal[];
+};
+
+export type ListMembershipModuleInputNamesType = [
+  "valueHash",
+  "list"
+];
+
+export type ListMembershipModuleOutputs = { isMember: CircuitSignal };
+
+export type ListMembershipModuleOutputNamesType = ["isMember"];
+
+export class ListMembership {
```

Comment:
Not important, but note that the pattern of putting helpers inside a class isn't required.  The helper here could probably be a free function.  I used the pattern for ProtoPODGPC because there were a lot of helpers grouped together, and I anticipated that there would someday be multiple families with similar helpers.  I'm pretty new to TypeScript so I don't know how standard that pattern might be.